timetabler/ssc/ssc_conn.py

```diff
@@ -191,7 +191,6 @@ def _auth(self, cwl_user, cwl_pass):
 
         # Submit request
         req2 = urllib2.Request(postURL2, data2)
-        resp2 = urllib2.urlopen(req2)
```

Comment:
The variable is unused but the request still has to be made to get the correct cookie (so much ugh with SSC auth).

You can amend to the following if you want :P

```diff
-        urllib2.urlopen(req2)
```